### ISSUE TITLE
[Broken Website] Any fix for this website ?

### ISSUE THREAD

**Issue by notenoughpizza** [darkreader/darkreader#2871]
**Website**
xcite.com

**How to Reproduce**
Turning on dark reader

**Expected behavior**
website should be working in dynamic theme

**Actual behavior**
dynamic theme is broken on the website, it's showing only headers with black colour but rest  of the page is bright white and i don't know how to fix it

**Screenshots**

**System info:**
OS: Windows 64 bit Pro, latest updates.
Browser, Version [83.0.4103.106](url) (Official Build) (64-bit)
Dark reader Version: 4.9.9
 
**Additional context**
i'd appreciate if the site can be fixed.

**Comment by notenoughpizza** [comment]
[
![Annotation 2020-06-16 032610](https://user-images.githubusercontent.com/66979448/84718656-d0fb6600-af81-11ea-9717-7216b45721c4.png)
](url)

**Comment by Gusted** [comment]
They use import URL's that don't allow external to see what they actually style is.

**Comment by notenoughpizza** [comment]
Yeah, it's a website for selling products mostly and it's the only site not working on dynamic for me, i have no clue to  fix it  by myself.

**Comment by Gusted** [comment]
Bug on our side, don't make a fix yet let first fix the `broken` style.

**Comment by notenoughpizza** [comment]
Thank you, appreciate your help
